app/models/energy/simulationparameter.py

```diff
@@ -0,0 +1,180 @@
+"""Simulation Parameter Schema"""
+from pydantic import BaseModel, Schema, validator, ValidationError
+from typing import List, Union
+from enum import Enum
+from app.models.common.datetime import Date
+
+
+class ReportingFrequency(str, Enum):
+    detailed = 'Detailed'
+    timestep = 'Timestep'
+    hourly = 'Hourly'
+    daily = 'Daily'
+    monthly = 'Monthly'
+    run_period = 'RumPeriod'
+    environment = 'Environment'
+    annual = 'Annual'
+
+
+class SimulationOutput(BaseModel):
+    """Lists the desired outputs from the simulation and the format in which to report them."""
+
+    type: Enum('SimulationOutput', {'type': 'SimulationOutput'})
+
+    reporting_frequency: ReportingFrequency = ReportingFrequency.hourly
+
+    include_sqlite: bool = Schema(
+        default=False
+    )
+
+    include_html: bool = Schema(
+        default=False
+    )
+
+    outputs: List[str] = Schema(
+        default=None
+    )
+
+    summary_reports: List[str] = Schema(
+        default=None
+    )
+
+
+class SimulationControl(BaseModel):
+    """Used to specify which types of calculations to run."""
+
+    type: Enum('SimulationControl', {'type': 'SimulationControl'})
+
+    do_zone_sizing: bool = Schema(
+        default=False
+    )
+
+    do_system_sizing: bool = Schema(
+        default=False
+    )
+
+    do_plant_sizing: bool = Schema(
+        default=False
+    )
+
+    run_for_run_periods: bool = Schema(
+        default=True
+    )
+
+    run_for_sizing_periods: bool = Schema(
+        default=True
+    )
+
+
+class SolarDistribution(str, Enum):
+    minimal_shadowing = 'MinimalShadowing'
+    full_exterior = 'FullExterior'
+    full_interior_and_exterior = 'FullInteriorAndExterior'
+    full_exterior_with_reflection = 'FullExteriorWithReflections'
+    full_interior_and_exterior_with_reflections = 'FullInteriorAndExteriorWithReflections'
+
+
+class CalculationMethod(str, Enum):
+
+    average_over_days_in_frequency = 'AverageOverDaysInFrequency'
+    timestep_frequency = 'TimestepFrequency'
+
+
+class ShadowCalculation(BaseModel):
+    """Used to describe settings for EnergyPlus shadow calculation."""
+
+    type: Enum('ShadowCalculation', {'type': 'ShadowCalculation'})
+
+    solar_distribution: SolarDistribution = SolarDistribution.full_exterior
+
+    calculation_frequency: int = Schema(
+        20,
```

Comment:
I know that the EnergyPlus default is 20 but we tend to use 30 in honeybee.  It's really a subjective decision but maybe we should make it 30 for consistency with what we usually do.